packages/sku/config/webpack/utils/loaders.js

```diff
@@ -91,18 +91,18 @@ const makeCssLoaders = (options = {}) => {
           },
         ]
       : []),
-
     {
       loader: require.resolve('less-loader'),
     },
   ];
 };
 
-const makeVanillaCssLoaders = (options = {}) => {
-  const { isProductionBuild, MiniCssExtractPlugin, browserslist } = options;
+const makeExternalCssLoaders = (options = {}) => {
+  const { target, isProductionBuild, MiniCssExtractPlugin, browserslist } =
+    options;
 
   return [
-    MiniCssExtractPlugin.loader,
+    ...(!target || target === 'browser' ? [MiniCssExtractPlugin.loader] : []),
```

Comment:
Curious what led to this change. Can you provide any context?

Reply:
This is the same behaviour as https://github.com/seek-oss/sku/blob/976fab0ecd97b02994a9d1fc6f60e1f1c51719e5/packages/sku/config/webpack/utils/loaders.js#L49

For VE files, we always use the loader (previous behaviour).
For non-VE files (plain CSS), we need the loader in the client build (`target` comes from [here](https://github.com/seek-oss/sku/blob/976fab0ecd97b02994a9d1fc6f60e1f1c51719e5/packages/sku/config/webpack/plugins/sku-webpack-plugin/index.js#L47)).